frontend/src/container/ListOfDashboard/SearchFilter/__tests__/utils.test.ts

```diff
@@ -0,0 +1,55 @@
+import { TOperator } from '../types';
+import { executeSearchQueries } from '../utils';
+
+describe('executeSearchQueries', () => {
+	const firstDashboard = {
+		id: 11111,
+		uuid: 'uuid1',
```

Comment:
can you please make this `v4` from `uuid` rather than constant string

Reply:
Yes :+1: